gulpfile.js

```diff
@@ -0,0 +1,12 @@
+var gulp = require('gulp');
+var sass = require('gulp-sass');
+
+gulp.task('sass', function () {
+  return gulp.src('./sass/baking-expeditions.scss')
+    .pipe(sass().on('error', sass.logError))
+    .pipe(gulp.dest('./public/stylesheets'));
+});
+
+gulp.task('sass:watch', function () {
```

Comment:
Does this "sass:watch" task start running when you run `gulp sass`?

Does the watch task run in the background while you are developing and automatically runs `gulp sass` when baking-expeditions.scss is changed?